clapi-hs.cabal

```diff
@@ -18,6 +18,7 @@ library
   exposed-modules:     Data.Map.Mos
                      , Data.Map.Mol
                      , Data.Map.Clapi
+                     , Clapi.EncodingProtocol
```

Comment:
`CodecProtocol`?

Reply:
To me that would imply a stream, but maybe this is SerialisationProtocol or something as it is the join of those 2 things?